.github/workflows/build-images.yaml

```
@@ -30,14 +30,25 @@ jobs:
         id: add-branch-tag
         run: |
           echo "IMG_TAGS=${GITHUB_REF_NAME/\//-} ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
-      - name: Set Authorino version
-        id: authorino-version
+      - name: Set Authorino build info
+        id: authorino-build-info
         run: |
           if [[ ${GITHUB_REF_NAME/\//-} =~ ^v[0-9]+\.[0-9]+\.[0-9]+(-.+)?$ ]]; then
-          echo "VERSION=${GITHUB_REF_NAME/\//-}" >> $GITHUB_ENV
+            echo "VERSION=${tag#v}" >> $GITHUB_ENV
```

I haven't tested it, but maybe...

```suggestion
          tag=${GITHUB_REF_NAME/\//-}
          echo "VERSION=${tag#v}" >> $GITHUB_ENV
```

---

I must have missed that part of the code... thanks!